scripts/adjustEnvVars.sh

```diff
@@ -165,14 +168,9 @@ MAKE="$MAKE1 --jobs=$JOBS"
 
 KERNEL_NAME=$(uname -s | tr '[:upper:]' '[:lower:]')
 
-function rmStep(){
-  rm -rf "$@"
-  rmdir -p --ignore-fail-on-non-empty `dirname "$@"`
```

Comment:
I didn't really get the point of using `rmdir` right after a `rm -rf`. `rm -rf` removes folders regardless of whether or not they're full/empty